### DISCUSSION TITLE
Apache NetBeans 19 release candidates

### DISCUSSION THREAD

**Discussion by neilcsmith-net** [apache/netbeans#6225] [General]
Release candidates for the next release of Apache NetBeans will be available weekly until release. Your help in testing, reporting and verifying issues would be highly appreciated by the whole Apache NetBeans community.

Changes from NetBeans 18 can be seen at https://github.com/apache/netbeans/releases/tag/19-rc1

## Binary / source zip

Binary and source zips are available from -

https://nightlies.apache.org/netbeans/candidate/netbeans/

## Installers

Unsigned Windows, Linux DEB and Linux RPM installers are available from -

https://nightlies.apache.org/netbeans/candidate/installers/

NB. We are working on making other installers available for release candidates from the same location.

## VSCode plugin

The VSCode plugin is available from - 

https://nightlies.apache.org/netbeans/candidate/vsix/

## IMPORTANT

These release candidates are pre-release software, and not official releases of Apache NetBeans.

**Comment by mbien** [comment]
known NB19 RCI issues with pending fixes for RC2 which might interfere with testing:
- https://github.com/apache/netbeans/issues/6214
- https://github.com/apache/netbeans/issues/6222

**Comment by neilcsmith-net** [comment]
Apache NetBeans 19-rc3 is now available from the links above. Changes from 19-rc1 can be seen at https://github.com/apache/netbeans/releases/tag/19-rc3

The eagle-eyed might notice we skipped announcing the build of 19-rc2! :smile:  This was to get the updated Gradle library into a release candidate as early as possible.

Both issues mentioned by @mbien above should now be resolved.

**Comment by neilcsmith-net** [comment]
Apache NetBeans 19-rc4 is now available from the links above. Changes from 19-rc3 can be seen at https://github.com/apache/netbeans/releases/tag/19-rc4

This will be our final release candidate unless high/critical priority issues or pull requests are flagged. Your help with testing is highly appreciated!

**Comment by neilcsmith-net** [comment]
Apache NetBeans 19-rc5 is now available from the links above. Changes from 19-rc4 can be seen at https://github.com/apache/netbeans/releases/tag/19-rc5  There are a small number of changes, primarily related to creation of new Gradle projects with the updated tooling.

This will be our final release candidate unless high/critical priority issues or pull requests are flagged. Your help with testing is highly appreciated!
